diff -urN wordinspect-0.1a.old/dict.c wordinspect-0.1a/dict.c
--- wordinspect-0.1a.old/dict.c	Sat Sep 19 05:31:41 1998
+++ wordinspect-0.1a/dict.c	Sun Sep  8 19:22:46 2002
@@ -27,6 +27,7 @@
 #include <unistd.h>
 #include <sys/types.h>
 #include <sys/wait.h>
+#include <string.h>
 #include <strings.h>
 #include <gtk/gtk.h>
 #include <stdio.h>
diff -urN wordinspect-0.1a.old/gladesig.c wordinspect-0.1a/gladesig.c
--- wordinspect-0.1a.old/gladesig.c	Sat Sep 19 05:31:44 1998
+++ wordinspect-0.1a/gladesig.c	Sun Sep  8 19:20:28 2002
@@ -170,12 +170,12 @@
       usage(argv[0]);
       exit(-1);
     }
-    do_define(NULL,word);
+    do_define(word);
   }
   else
   {
     if (!word) word="";
-    do_find(NULL,word);
+    do_find(word);
   }
 /*   winWordFinder = create_winWordFinder (); */
 /*   gtk_widget_show (winWordFinder); */
@@ -494,13 +494,13 @@
   return TRUE;
 }
 
-gint do_define(GtkWidget *w, gpointer user_data)
+gint do_define(gpointer user_data)
 {
   define_word((gchar *)user_data);
   return TRUE;
 }
 
-gint do_find(GtkWidget *w, gpointer user_data)
+gint do_find(gpointer user_data)
 {
   GtkWidget *winWordFinder;
   
diff -urN wordinspect-0.1a.old/gladesig.h wordinspect-0.1a/gladesig.h
--- wordinspect-0.1a.old/gladesig.h	Sat Sep 19 05:31:45 1998
+++ wordinspect-0.1a/gladesig.h	Sun Sep  8 19:20:18 2002
@@ -39,8 +39,8 @@
 int on_labelDef_button_release_event_before(GtkWidget *w, gpointer user_data);
 int define_word(char *word);
 gint do_menu(GtkWidget *w, GdkEventButton *bevent, const char *word);
-gint do_define(GtkWidget *w, gpointer user_data);
-gint do_find(GtkWidget *w, gpointer user_data);
+gint do_define(gpointer user_data);
+gint do_find(gpointer user_data);
 void selection_received_search (GtkWidget *widget,
                                 GtkSelectionData *selection_data,
                                 gpointer data);
diff -urN wordinspect-0.1a.old/utils.c wordinspect-0.1a/utils.c
--- wordinspect-0.1a.old/utils.c	Sat Sep 19 05:31:45 1998
+++ wordinspect-0.1a/utils.c	Sun Sep  8 19:21:55 2002
@@ -25,6 +25,7 @@
 
 #include <stdio.h>
 #include <stdlib.h>
+#include <string.h>
 #include <strings.h>
 #include <stdarg.h>
 #include "utils.h"
